packages/terra-demographics-banner/src/DemographicsBanner.module.scss

```diff
@@ -106,5 +109,10 @@
     .identifiers {
       text-align: right;
     }
+
+    .row {
+      flex: 100%;
+      order: 3;
```

Comment:
Why was an order of 3 selected?  Should we add a comment so that the reason for using this value is clear?

Reply:
Good point, will add a comment. This is needed to "undo" the changing of the order of elements to fix the order read by the screen reader back to its original visual ordering, since the DOM ordering impacts screen readers but not the css. In this case, the 'row' element should be the third item in the flexbox, when it would be the second by default based on the order of elements.